cli/test/commands/create-mint/index.test.ts

```diff
@@ -1,24 +1,24 @@
 import { expect, test } from "@oclif/test";
 import { initTestEnvIfNeeded } from "../../../src/utils/initTestEnv";
-import { getPayer } from "../../../src";
+import { defaultSolanaWalletKeypair, getPayer } from "../../../src";
 import { requestAirdrop } from "../../helpers/helpers";
-
+import { Keypair } from "@solana/web3.js";
+import { bs58 } from "@coral-xyz/anchor/dist/cjs/utils/bytes";
 describe("create-mint", () => {
   test.it(async () => {
     await initTestEnvIfNeeded();
-    await requestAirdrop(getPayer().publicKey);
-
-    const mintAuthority = getPayer().publicKey.toBase58();
     const mintDecimals = 5;
-    const mintKeypair = getPayer();
-
+    const mintKeypair = defaultSolanaWalletKeypair();
```

Comment:
using the solana wallet as mintkeypair is unintended (this would cause the user's pubkey to become a Mint address), but i think its non critical. we can add more test cases in followup prs imo.